e2e-tests/gef/cypress/fixtures/mocks/mock-deals.js

```diff
@@ -176,12 +175,93 @@ export const MOCK_APPLICATION_AIN = {
     status: 'IN_PROGRESS',
     requiredFields: [],
   },
+  ukefDecision: [],
   ...commonApplicationDetails,
 };
 
 export const MOCK_APPLICATION_MIN = {
   status: DEAL_STATUS.UKEF_ACKNOWLEDGED,
   submissionType: DEAL_SUBMISSION_TYPE.MIN,
+  ukefDecisionAccepted: true,
+  eligibility: {
```

Comment:
can we create `MOCK_ELIGIBILITY`? To be DRY

Reply:
done